Site/SiteAnalyticsModule.php

```diff
@@ -146,11 +175,98 @@ public function hasAnalytics()
 		return (
 			$this->hasGoogleAnalytics() ||
 			$this->hasFacebookPixel() ||
+			$this->hasTwitterPixel() ||
 			$this->hasBingUET()
 		);
 	}
 
 	// }}}
+	// {{{ public function displayAnalytics()
+
+	public function displayAnalytics()
+	{
+		$this->displayGoogleAnalytics();
+		$this->displayFacebookPixelImage();
+		$this->displayTwitterPixelImages();
+		$this->displayBingUETImage();
+	}
+
+	// }}}
+	// {{{ public function getInlineJavascript()
+
+	public function getInlineJavascript()
+	{
+		$js = '';
+
+		if ($this->hasFacebookPixel()) {
+			$js.= $this->getFacebookPixelInlineJavascript();
+		}
+
+		if ($this->hasBingUET()) {
+			$js.= $this->getBingUETInlineJavascript();
+		}
+
+		if ($this->hasGoogleAnalytics()) {
+			$js.= $this->getGoogleAnalyticsInlineJavascript();
```

Comment:
We repeat this inline JS in `displayGoogleAnalytics()`. It should probably be removed from one of the methods.